.all-contributorsrc

```diff
@@ -906,6 +906,32 @@
         "translation",
         "tutorial"
       ]
+    },
+    {
```

Comment:
You're showing up twice here. We only need one 😄

Reply:
oops, I changed it. Sorry I had not see it 🙈